keel-core/src/main/kotlin/com/netflix/spinnaker/keel/persistence/ResourceRepository.kt

```diff
@@ -123,6 +134,7 @@ interface ResourceRepository : PeriodicallyCheckedRepository<ResourceHeader> {
       history.isActuating() -> ACTUATING
       history.isError() -> ERROR
       history.isCreated() -> CREATED
+      history.isPaused() -> PAUSED
```

Comment:
What state do you want the UI to show if the resource was paused, but has just been resumed? Right now the state is `UNKNOWN` for this case. Is that appropriate?

Reply:
my 2 cents: I'd like to avoid `UNKNOWN` wherever possible. It's scary and people won't like it.

maybe bad idea: what if we had a `RESUMED` status that allowed the UI to show a message like "Management was paused, but you just resumed it. Spinnaker will check the the current status and might take action to resolve any drift from the declarative configuration"?

Reply:
I agree, and I think a `RESUMED` status is fine for now.

Reply:
Hmm, but to your point earlier @emjburns, that status will only last a few seconds since another actuation-related event will be emitted immediately after. This is why I did not add a new status. Am I missing something?

Reply:
While that's true, I think the idea is philosophically what I mentioned before: `UNKNOWN` is a very scary thing to see, so when we run into a case where it's possible to avoid an `UNKNOWN` status showing up in the first place I'd advocate we avoid the risk of a scary status even if it's a small risk.

Probably not going to cause huge problems in this specific case either way, but I think it's wise to build up the habit/muscle of keeping UNKNOWN states as rare as possible.

Reply:
Fair enough. Let me add that status.